src/main/scala/org/broadinstitute/hail/expr/package.scala

```diff
@@ -3,7 +3,7 @@ package org.broadinstitute.hail
 package object expr {
   type SymbolTable = Map[String, (Int, BaseType)]
 
-  type Aggregator = (() => Any, (Any) => Any, (Any, Any) => Any, Int)
+  type Aggregator = (() => Any, (Any, Any) => Any, (Any, Any) => Any, Int)
```

Comment:
You might switch the names and make the typed version `Aggregator`.  I thought this a very reasonable object and, for example, the VSM aggregation functions should take an aggregator.  (But don't add that to this PR.)